esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -412,6 +418,71 @@ void EspHttpBinding::readUnrestrictedResources(const char* resources)
     }
 }
 
+bool EspHttpBinding::canRedirectAfterAuth(const char* url)
+{
+    if (isEmptyString(url))
+        return false;
+
+    bool* found = invalidURLsAfterAuth.getValue(url);
+    return (!found || !*found);
+}
+
+bool EspHttpBinding::isCORSRequest(const char* originHeader)
```

Comment:
const method?

Reply:
Not this one because it calls getPort().